### DISCUSSION TITLE
Enable Folding on package editor for Oracle

### DISCUSSION THREAD

**Discussion by AlexuLag** [dbeaver/dbeaver#13137] [Ideas]
Hi, great job with DBeaver for oracle, I been using this tool the last year and is an amazing alternative for oracle sqldeveloper and Toad, I want to ask you if there is any plan for adding support to folding and unfolding  code , and bookmarks support in package view

**Comment by flambchap** [comment]
Hi. Now we do have such plans :) Thank you for your idea and your kind words.
